testgrid/config/config.yaml

```diff
@@ -1925,6 +1925,8 @@ dashboards:
   dashboard_tab:
     - name: daisy-e2e
       test_group_name: ci-daisy-e2e
+      open_test_template:
```

Comment:
To be safe, you probably also want 'results_text' and 'results_url_template' (open_test_template is for clicking on the cell directly, these are for when you right-click on a cell and want to open the target).

Reply:
Decided to leave these alone for now, hoping to keep a way to link back to Gubenator's test run summary page.